src/stylesheets/components/_search.scss

```diff
@@ -11,69 +11,68 @@ $usa-btn-big-width:     11.6rem;
 
 .usa-search {
   @include clearfix;
-  max-width: none;
+  @include typeset;
   position: relative;
 
+  [role=search] {
+    display: flex;
+  }
+
   [type=submit],
   .usa-search-submit {
```

Comment:
Do we actually use `.usa-search-submit`? I don't see it.

Reply:
OK removed in 03c8a43, maybe we thought it would be good to have both a class and this at one point.